packages/@glimmer/runtime/lib/component/manager.ts

```diff
@@ -15,8 +15,8 @@ import { MINIMAL_CAPABILITIES } from './interfaces';
 import { Reference, UNDEFINED_REFERENCE } from '@glimmer/reference';
 
 // TODO: This manager appears to be unused
```

Comment:
🤷‍♂️ remove it?

Reply:
👍

Reply:
Actually it is now used by the benchmark-env it seems, I'll update the comment, we should remove it in the next set of refactors